crates/turbo-tasks-macros/src/lib.rs

```diff
@@ -166,6 +166,13 @@ pub fn function(args: TokenStream, input: TokenStream) -> TokenStream {
     function_macro::function(args, input)
 }
 
+#[allow_internal_unstable(min_specialization, into_future, trivial_bounds)]
+#[proc_macro_error]
+#[proc_macro_attribute]
+pub fn test_tt(_args: TokenStream, input: TokenStream) -> TokenStream {
+    derive::derive_value_debug(input)
+}
+
```

Comment:
Is this left over from your testing? This doesn't look like it's called anywhere in the stack of PRs?